src/css/controls/states.less

```diff
@@ -218,8 +218,11 @@ body .jwplayer.jw-state-error,
 
     .jw-settings-menu {
         height: 100%;
-        top: 0;
-        bottom: 0;
+        top: 50%;
```

Comment:
Setting all sides to 50% is redundant, unless it's intended to cascade over previous declarations. Since `top` and `left` are 50%, `bottom` and `right` can both be `auto` or probably just left out here.

Reply:
dope.js